english/plugh/outside/outside1.md

```diff
@@ -0,0 +1,10 @@
+You decided to rest a little bit. 
+
+Password is still in your pocket. Weather is nice, birds are tweeting. 
+
+*Pstst*. Can of soda is opened. You drink slowly and enjoy it. 
+
+However your inner voice stars raising up. You think you noticed some movement. 
+
+[Look around](outside/outside1/outside.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.